reference/configuration/framework.rst

```diff
@@ -1734,7 +1734,9 @@ must implement the :class:`Symfony\\Component\\Translation\\Formatter\\MessageFo
 **type**: ``array`` **default**: ``[]``
 
 This option allows to define an array of paths where the component will look
-for translation files.
+for translation files. Translation entries from later directories overwrite
+earlyer ones. The last processed dir is `default_path <reference-translator-default_path>`
+(which has a default value).
```

Comment:
This order is the result of my tests. Could somebody confirm please?

In case you do not use flex: `app/Resources/translations` is after `default_path`.

Reply:
Yes, it's correct.